src/js/OTHelpers.coffee

```diff
@@ -12,13 +12,15 @@ getPosition = (divName) ->
   pubDiv = document.getElementById(divName)
   if !pubDiv then return {}
   computedStyle = if window.getComputedStyle then getComputedStyle(pubDiv, null) else {}
+  transform = new WebKitCSSMatrix(window.getComputedStyle(pubDiv).transform)
```

Comment:
Looks like this breaks on Android for elements that don't have a transform set. (Tested on Android 4.4 without XWalk).

Workaround would be : 
```
transform = new WebKitCSSMatrix(window.getComputedStyle(pubDiv).transform || "")
```